app/src/main/java/com/example/githubapp/repository/ProfileRepository.kt

```diff
@@ -0,0 +1,27 @@
+package com.example.githubapp.repository
+
+import com.example.githubapp.api.GitHubApi
+import com.example.githubapp.models.Repos
+import com.example.githubapp.models.UserData
+import dagger.Module
+import dagger.Provides
+import io.reactivex.rxjava3.core.Observable
+import io.reactivex.rxjava3.core.Observer
+import retrofit2.Retrofit
+import javax.inject.Inject
+
+@Module
+class ProfileRepository @Inject constructor(var retrofit: Retrofit) {
```

Comment:
Is it a module or a repository? A module shouldn't be injected and a repository shouldn't have `@Module` annotation.